src/components/artwork/save.tsx

```diff
@@ -6,10 +6,11 @@ import Icon from "../icon"
 import styled from "styled-components"
 import colors from "../../assets/colors"
 
-const { CURRENT_USER } = require("sharify").data
+import * as Artsy from "../../components/artsy"
```

Comment:
I’m gonna keep this as-is, I like including the `Artsy` namespace, `ContextProps` on its own is a bit ambiguous.